Formula/walk.rb

```diff
@@ -0,0 +1,26 @@
+class Walk < Formula
+  desc "Terminal navigator"
+  homepage "https://github.com/antonmedv/walk"
+  url "https://github.com/antonmedv/walk/archive/refs/tags/v1.4.0.tar.gz"
+  sha256 "301f5dff2b8f27595a08a12626125d28313b5a92e97ccdaf4d0cb89111ebab9a"
```

Comment:
```suggestion
  sha256 "606568e098a91c7b0e694534bf0d268d92cc9ac19bce9e4671775837176eb9f5"
```

Reply:
what about the sha?

Reply:
Trying to figure out. I updated in and pushed.

Reply:
Done.